code/about.html

```diff
@@ -0,0 +1,113 @@
+      <p><a href="https://www.instagram.com/textwork_ch/" target="_blank" rel="noopener noreferrer"><img class="inst-sm" src="../img/glyph-logo_May2016.png" alt="textwork_ch"></a></p>
+      <p><a href="https://www.linkedin.com/in/jaksivio/" target="_blank" rel="noopener noreferrer"><img class="linkd-sm" src="../img/In-Black-28px-R.png" alt="jaksivio"></a></p>
+    </div>
+  </section>
+
+
+  <nav class="navcontainer">
+    <div class="navbar">
+      <ul>
+        <li><a href="./services.html">Services</a></li>
+        <li><a class="active" href="./about.html">About</a></li>
+        <!---<li><a href="./portfolio.html">Portfolio</a></li>-->
+        <li><a class="no-link" href="https://www.textwork.ch/"><img class="logo" src="../img/textwork_blogo.png" alt="Textwork-logo"></a></li>
+        <li><a href="./blog.html">Blog</a></li>
+        <li><a href="./contact.html">Contact</a></li>
+      </ul>
+    </div>
+  </nav>
+  <header class="bg-about">
+    <h1 class="headerh1">About</h1>
+  </header>
+  
+
+  <div class="content">
+    <section class="samerow-content">
+      <div class="flexchild-content 1">
+        <div class="img-violeta">
+          <img src="../img/IMG_1453-bw.jpg" alt="Violeta Jaksic">
+        </div>
+      </div>
+      <div class="flexchild-content 2">
+        <h3>Hi. I’m Violeta Jaksic.</h3>
+        <p>I’m a passionate geeky writer with a head full of ideas to help you stand out from the crowd with everything you write. I’m located in Switzerland and have a degree in technical communication.</p>
+        <p><b>My biggest talent?</b></p>
+        <p>Look beyond the rules. If you change your perspective, you suddenly see things in a completely different light.</p>
+      </div>
+    </section>
+    
+    <h1>Skill Set</h1>
+    <p>I am a <b>results-driven and self-motivated</b> professional with a broad range of experience in leading various operations relating to <b>technical writing and document management</b>. I am a dynamic <b>multilingual communicator</b> who is driven by challenges, undeterred by obstacles, and committed to furthering standards of excellence. I have proven success in gathering, reviewing, analyzing, creating, editing, and publishing <b>technical as well as creative content</b> within budgetary and time constraints.</p>
+    <p>I excel at uncovering anomalies, determining root causes, and proposing solutions to facilitate corrective actions. While others take pride in meeting all standards set before them, my passion is <b>delivering a service that goes beyond organizational expectations</b>, creating a win-win scenario for everyone involved. As a cross-functional and collaborative leader, I deliver integrated solutions that encourage long-term business growth and profitability.</p>
+
+
+
+
+  </div>
+
+
+
+
+  <footer class="bg-connect">
+    <h1 class="headerh1"><a href="./contact.html">Let's Connect</a></h1>
+  </footer>
+
+
+  <div class="footer">

```

Comment:
Try to name your classes/selecting appropriate tags to not confuse. I think the `<footer class="bg-connect">` should be a normal `<div>` and set the `<div class="footer">` to `<footer>`. 😊

The header, footer and main-tags are kind of special, and should be used in a correct way to not confuse SEO/accessibility (screen readers etc.).